ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPanel.java

```diff
@@ -1236,6 +1237,16 @@ public void onResponseReceived( Request request, Response response ) {
             refresh();
           }
         }
+
+        private String getErrorMessage( JsJob job, String state ) {
+          String errorMessage = "- " + job.getJobName() + ": " + Messages.getString( "bulkDeletePermissionDeniedError" );
```

Comment:
Here I think it's best to leave all the strings in the message properties and use "{0}" or similar to replace the job name. Something like this: "- {0} bulk delete msg" in the properties file.